en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+# Develop an API Proxy from a GitHub Repository Source
```

Comment:
Make sure to follow the .md file naming convention https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting#file-naming-convention
Need to rename this file:
`develop-a-git-api-proxy.md` to `develop-an-api-proxy-from-a-github-repository-source.md`